Core/HLE/proAdhoc.h

```diff
@@ -32,6 +32,7 @@
 #include "Core/HLE/sceKernel.h"
 #include "Core/HLE/sceKernelMutex.h"
 #include "Core/HLE/sceUtility.h"
+#include "UI/ChatScreen.h"
```

Comment:
We shouldn't include UI from inside Core.  The best option is to add checking in the `ChatScreen` class, inside its `update()` method, which is called each frame.  Since the count won't be drawn until then anyway, no need to update more frequently.

It could call a `GetChatCount()` function, or something, and then you wouldn't need to abuse the `g_Config` object for that value (which isn't really configuration.)  Also, `setChatPointer` would probably be gone, making managing the lifetime of that object completely UI's job - which is simpler.

-[Unknown]